prow/config.yaml

```diff
@@ -6107,6 +6107,19 @@ periodics:
       - "--repo=k8s.io/autoscaler=master"
       - "--root=/go/src"
       - "--timeout=60"
+      env:
+      - name: DOCKER_IN_DOCKER_ENABLED
+        value: true
+      # docker-in-docker needs privileged mode
+      securityContext:
+        privileged: true
+      volumeMounts:
+      - name: docker-graph
+        mountPath: /docker-graph
+    volumes:
+    - name: docker-graph
+      hostPath:
+        path: /mnt/disks/ssd0/docker-graph
```

Comment:
You either need to use an emptyDir here or set port 9999 to limit to one job per node since sharing docker isn't safe.
Making this better is on my todo O(soon)
@krzyzacy we should make these presets

Reply:
how do you want to deal with port, add it to our preset?